### PR TITLE
Skip failing Windows tests

### PR DESCRIPTION
**Pull request by Blast545**
Signed-off-by: Jorge Perez jjperez@ekumenlabs.com

First step towards https://github.com/ignitionrobotics/ign-gazebo/issues/1175

## Summary

As the title says, skips Windows tests for `ign-gazebo5`. Will get green Windows buildfarm before merging.

## Checklist
- [x] Signed all commits for DCO
- [x] Added tests (actually removing tests)
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Comment by Blast545** [comment]
Worskpace temp error on the Windows pr job:
* WIndows [Build Status](https://build.osrfoundation.org/job/ign_gazebo-pr-win/3384/)

Manual rebuild:
* WIndows [Build Status](https://build.osrfoundation.org/job/ign_gazebo-pr-win/3384/)

File: src/systems/physics/EntityFeatureMap_TEST.cc [hunk 1]
```diff
@@ -92,7 +92,8 @@ class EntityFeatureMapFixture: public InternalFixture<::testing::Test>
   public: EnginePtrType engine;
 };
 
-TEST_F(EntityFeatureMapFixture, AddCastRemoveEntity)
+// See https://github.com/ignitionrobotics/ign-gazebo/issues/1175
+TEST_F(EntityFeatureMapFixture, IGN_UTILS_TEST_ENABLE_ONLY_LINUX(AddCastRemoveEntity))
```
Comment (chapulina) [inline src/systems/physics/EntityFeatureMap_TEST.cc:96]: Can this be disabled only on Windows?

**Review by chapulina — COMMENTED** [review]
Nice! This is going in a good direction! There are some compilation and codecheck errors though

File: src/systems/physics/EntityFeatureMap_TEST.cc (hunk 1 above)
Comment (Blast545) [reply src/systems/physics/EntityFeatureMap_TEST.cc]: Sure, my bad there. Addressed here: 5901c2b

**Comment by Blast545** [comment]
* WIndows [Build Status](https://build.osrfoundation.org/job/ign_gazebo-pr-win/3395/)

**Comment by Blast545** [comment]
@ros-pull-request-builder retest this please

**Comment by chapulina** [comment]
> `@ros-pull-request-builder` retest this please

Whaaat did that work? Are @osrf-jenkins and @ros-pull-request-builder doing exactly the same thing? :scream:

**Comment by Blast545** [comment]
@chapulina I actually tried and wasn't expecting it to work, I was surprised in a good way.

**Comment by Blast545** [comment]
There seems to be an error with the PR jobs. For some reason, at least for this PR, after I add commits to retrigger the PR logic it tries to test against `gazeboMain` instead of `gazebo5`. However, the "retest please" works fine.

**Comment by Blast545** [comment]
@ros-pull-request-builder retest this please

**Comment by Blast545** [comment]
The PR job is not showing anymore failures for Windows, I would say this is ready for merge:
https://build.osrfoundation.org/job/ign_gazebo-pr-win/3420/

It has 100+ warnings though. I think we can iterate on those after #990 gets merged.

**Review by chapulina — APPROVED** [review]
Thanks for all the effort, @Blast545 !

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-01-24-citadel-edifice-fortress/1241/1

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-04-13-fortress-edifice/1367/1

**Pull request merged**